master/internal/command/ntsc.go

```diff
@@ -0,0 +1,29 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/determined-ai/determined/master/pkg/actor"
+)
+
+// MessageNTSC sends a msg to all ntsc managers.
+// CHECK: Tell(diff semantics?) or Ask
+func MessageNTSC(system *actor.System, msg interface{}) actor.Responses {
+	// CHAT: this could also accept an *actor.Context.
+	refs := []*actor.Ref{
+		// system.Get(actor.Addr(CommandActorPath)),
```

Comment:
are these all supposed to be commented out?

Reply:
for now yes but I plan on adding them in this pr. those can only be added after I add the message handlers to those actors and I waiting on that to make sure we chat about the other discussion points I left inline w/ the code changes first

Reply:
- fail the workspace deletion on error from termination
- panic if ntsc managers arent' up?